dev/com.ibm.ws.request.probes/src/com/ibm/wsspi/requestContext/RequestContext.java

```diff
@@ -193,4 +194,8 @@ public String toString() {
 		return "RequestContext [threadId=" + threadId + ", requestId="
 				+ requestId + ", state=" + state + "]";
 	}
+	
+	public boolean getisSlow() {
+		return isSlow;
+	}
```

Comment:
please move this method before toString to keep it with the other field access methods.  please also add setSlow(boolean b) method